README.md

```diff
@@ -40,20 +105,276 @@ conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults cugraph
 
 Note: This conda installation only applies to Linux and Python versions 3.6/3.7.
 
+
+
 <a name="source"></a>
 
+### Build from Source 
+

```

Comment:
I thought this was supposed to be in `CONTRIBUTING.md` now? cuDF and cuML already adopted the `CONTRIBUTING.md` solution for build and test instructions.

Reply:
Currently there is no link to Contributing, and that text does not explain how to build.
This is another punt to v0.8 :(

Reply:
oops, was looking at 0.8 branch